railties/lib/rails/generators/rails/app/templates/config/initializers/callback_terminator.rb

```diff
@@ -1,4 +1,5 @@
 # Be sure to restart your server when you modify this file.
 
-# Do not halt callback chains when a callback returns false.
+# Do not halt callback chains when a callback in Active Record
+# or Active Model returns false.
```

Comment:
I don't think this should mention Active Record or Active Model. This is for users who don't know Active Support callbacks don't have the same terminator. And the mention just makes that confusing. They can turn to the upgrading guide for an explanation.